src/Stryker.CLI/Stryker.CLI/CommandLineConfigHandler.cs

```diff
@@ -125,6 +126,8 @@ private void PrepareCliOptions(IStrykerInputs inputs)
             AddCliInput(inputs.ProjectUnderTestNameInput, "project", "p", argumentHint: "project-name.csproj");
             AddCliInput(inputs.MutationLevelInput, "mutation-level", "l");
 
+            AddCliInput(inputs.TestCaseFilterInput, "filter", "");
```

Comment:
I think filter is too generic. How about testCaseFilter. Also I'm not sure this needs to be a command line option and it probably should be a file based option. See https://github.com/stryker-mutator/stryker-net/blob/v1.0/src/Stryker.CLI/Stryker.CLI/FileBasedInput.cs and https://github.com/stryker-mutator/stryker-net/blob/v1.0/src/Stryker.CLI/Stryker.CLI/JsonConfigHandler.cs

Reply:
Thank you for your feedback.

Agreed to renaming `filter` option to `testCaseFilter`.

I'll remove the command line option and place it to the file based options.

Reply:
The option has been moved from command line to file, and renamed to `test-case-filter`. I used the dashes instead of camel case to be consistent with other options naming style.